.github/workflows/ci.yml

```diff
@@ -10,11 +10,13 @@ jobs:
     strategy:
       matrix:
         emacs_version: [25, 26, 27, 28]
+        make_target: ['checkdoc', 'longlines', 'test', 'compile']
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: CI
         env:
           VERSION: ${{ matrix.emacs_version }}
+          TARGET: ${{ matrix.make_target }}
         run: >-
-          make docker CMD="make -k compile checkdoc longlines test"
+          make docker CMD="make -k $TARGET VERSION=$VERSION"
```

Comment:
I think this will make it so we end up building the docker image 16 times instead of only 4 times, which might make CI slower.

Reply:
See https://github.com/radian-software/prescient.el/pull/157

Reply:
I've realized that I was focusing on the wrong thing when I wrote that. I've closed that PR.